android/src/main/java/com/thebluealliance/androidclient/binders/EventInfoBinder.java

```diff
@@ -169,7 +169,7 @@ public void updateData(@Nullable Model data) {
         eventYoutubeTitle.setText(mActivity.getString(R.string.view_event_youtube, data.eventKey));
         eventYoutubeContainer.setOnClickListener(mSocialClickListener);
 
-        eventCdContainer.setTag("http://www.chiefdelphi.com/media/photos/tags/" + data.eventKey);
+        eventCdContainer.setTag("https://www.chiefdelphi.com/search?q=category%3A11%20tags%3A" + data.eventKey);
```

Comment:
Yeesh, that's a janky way to pass a URL around.

Reply:
janky is the only way I know how :(